### PR TITLE
Feature/us5985 adventure club

### PR DESCRIPTION
**Pull request by timmyg**

### CODE REVIEW

File: front_end/src/app/admin/events/rooms/room-group-list.component.html [hunk 1]
```diff
@@ -33,7 +33,8 @@
         <div class="row">
           <strong class="col-sm-4 col-md-2 push-half-top">Set as Adventure Club</strong>
           <span class="col-sm-8 col-md-10 adventure-club-toggle">
-            <input class='tgl tgl-light' id='isAdventureClub' type='checkbox' [(ngModel)]="isAdventureClub">
+            <!-- <input class='tgl tgl-light' id='isAdventureClub' type='checkbox' [checked]="isAdventureClub" (change)="onAcChange()"> -->
```
Comment (jpc11623) [inline front_end/src/app/admin/events/rooms/room-group-list.component.html]: Can the commented code be removed?

File: front_end/src/app/admin/events/rooms/room-group-list.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    const clickEvent = {target: { checked: true } };
+    it('should toggle eventToUpdate if no groups active', () => {
+      // spyOn(fixture, 'isAdventureClub').and.returnValue(false);
+      fixture.isAdventureClub = false;
+      spyOn(fixture, 'hasActiveRooms').and.returnValue(false);
+      fixture.toggleAdventureClub(clickEvent);
+      expect(fixture.isAdventureClub).toBeTruthy();
+    });
+    it('should return error and not allow if groups active ', () => {
+      // spyOn(fixture, 'isAdventureClub').and.returnValue(false);
```
Comment (jpc11623) [inline front_end/src/app/admin/events/rooms/room-group-list.component.spec.ts]: Ditto

File: front_end/src/app/admin/events/rooms/room-group-list.component.ts [hunk 3]
```diff
@@ -49,6 +63,16 @@ export class RoomGroupListComponent implements OnInit {
 
   }
 
+  // get the adventure club event if AdventureClub is true for room, else return service event
+  setCurrentEvent(isAdventureClub: boolean) {
+    this.isAdventureClub = isAdventureClub;
+    if (this.isAdventureClub) {
+      this.eventToUpdate = _.filter(this.eventsMap, {'EventTypeId': 20})[0];
```
Comment (jpc11623) [inline front_end/src/app/admin/events/rooms/room-group-list.component.ts]: Do we have the AC event type id in a constants file somewhere?

File: SignInCheckIn/MinistryPlatform.Translation/Repositories/EventRepository.cs [hunk 4]
```diff
@@ -37,15 +37,20 @@ public class EventRepository : IEventRepository
...
                 "Event_End_Date",
                 "[Early_Check-in_Period]",
                 "[Late_Check-in_Period]",
                 "Event_Type_ID_Table.Event_Type",
+                "Events.Event_Type_ID",
                 "Congregation_ID_Table.Congregation_Name",
                 "Events.Congregation_ID",
-                "Congregation_ID_Table.Location_ID"
+                "Congregation_ID_Table.Location_ID",
+                "[Allow_Check-in]"
```
Comment (dtkocher) [inline SignInCheckIn/MinistryPlatform.Translation/Repositories/EventRepository.cs]: you can remove []

**Pull request merged**